new-packaging/README.md

```diff
@@ -0,0 +1,21 @@
+You need `xmlstarlet`, `dos2unix`
+
+From `/path/to/neo4j` root directory run `mvn install`
+
+`mvn clean install -DskipTests`
+
+you need `cypher-shell` packaged
+
+`make out/cypher-shell.zip`
```

Comment:
It's not clear that this means you should clone cypher-shell and build it. Instead, suggest people download the zip file from teamcity maybe.